packages/netlify-cms-core/src/components/Editor/EditorToolbar.js

```diff
@@ -273,7 +273,7 @@ export default class EditorToolbar extends React.Component {
       isNewEntry,
     } = this.props;
     if (currentStatus) {
-      return [
+      return (<>
```

Comment:
Fragments are more appropriate here -- we don't actually have a list of items (which would need `key`s).